src/css/modules/buttons.scss

```diff
@@ -79,7 +79,13 @@
       font-size: 16px;
       padding: 14px 24px 10px;
       margin: 0 auto;
+      max-width: 184px;
+
+      &:nth-child(n) {
```

Comment:
Please, check is it a valid rule.

I believe `&:nth-child(n) {}` can be replaced with '& > {}'